### PR TITLE
Vonage official support

### PR DESCRIPTION
**Pull request by jeffswartz**
### Contributing checklist
- [√] Code must follow existing styling conventions
- [√] Added a descriptive commit message
- [ ] Sample apps updated if needed

### Solves issue(s)
(none)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,6 @@
 
 React Native library for using [OpenTok](https://tokbox.com/developer/).
 
-**Important:** Please note that this library is not officially supported by Vonage.
-
```
Comment (jeffswartz) [inline README.md:8]:
```suggestion
This library is now officially supported by Vonage.

```

File: README.md [hunk 2]
```diff
@@ -4,8 +4,6 @@
 
 React Native library for using [OpenTok](https://tokbox.com/developer/).
```
Comment (jeffswartz) [inline README.md:6]:
```suggestion

This library is now officially supported by Vonage.

```

File: README.md [hunk 3]
```diff
@@ -4,8 +4,6 @@
...
 
-**Important:** Please note that this library is not officially supported by Vonage.
```
[the remark this answers is not in the document]
Comment (jeffswartz) [reply README.md:7]: Done.

File: README.md [hunk 4]
```diff
@@ -4,8 +4,6 @@
...
-
 - [Pre-Requisites:](#pre-requisites)
```
[the remark this answers is not in the document]
Comment (jeffswartz) [reply README.md]: I agree. I removed the unneeded TOC links.

**Review by kapilmishr — APPROVED** [review]

**Comment by flogy** [comment]
Awesome 🎉🎉🥳

**Pull request merged**